subsys/nrf_security/Kconfig.tls

```diff
@@ -256,6 +260,20 @@ config MBEDTLS_SSL_KEEP_PEER_CERTIFICATE
 	  reduces RAM usage.
 	  Corresponds to MBEDTLS_SSL_KEEP_PEER_CERTIFICATE in mbed TLS config file.
 
+config MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE
+	bool
+	prompt "Give acces to peer certificate after handshake"
+	default n
+	help
+	  Not sure
+
+config MBEDTLS_SSL_HANDSHAKE_WITH_PSK_ENABLED
+	bool
+	prompt "Allow Handshake with psk"
+	default n
+	help
+	  Not sure
```

Comment:
:smile:

Reply:
Can the PR be put in draft mode for now then if it makes sense?